modules/compiler/utils/CMakeLists.txt

```diff
@@ -53,7 +53,30 @@ if(EXISTS "${CA_LLVM_INSTALL_DIR}/lib/cmake/lld/LLDConfig.cmake")
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   )
 
+  # Explicitly link against the individual static versions of the lld
```

Comment:
Before we checked if LLD was enabled - should we do that here?

Reply:
We checked by `if(EXISTS "${CA_LLVM_INSTALL_DIR}/lib/cmake/lld/LLDConfig.cmake")`. This new logic I added is inside that `if`, so that should be handled already, this should do nothing when LLD is disabled.